api-service/crowdsource-webapp/src/app/relative-toxicity-job/relative-toxicity-job.component.ts

```diff
@@ -51,19 +55,21 @@ export class RelativeToxicityJobComponent extends BaseJobComponent<RelativeToxic
   commentBIndex = 0;
 
   public buildAnswer(): {} {
-    if (this.useRadioButtons) {
-      return { toxicityOption: this.toxicityOption };
-    } else {
-      return { toxicityOption: this.getToxicityOption() };
-    }
+    return {
+      comment_a: this.question.comment_a,
+      comment_b: this.question.comment_b,
+      id_a: this.question.id_a,
+      id_b: this.question.id_b,
+      toxicityOption: this.useRadioButtons ? this.toxicityOption : this.getToxicityOption()
+    };
   }
 
   protected updateQuestion(): void {
     super.updateQuestion();
     // TODO: The data is incorrectly stored in the db as a JSON string instead
     // of JSON, like it is for the other jobs. We should fix this and then
     // remove the JSON.parse() here.
-    this.question = JSON.parse(this.question as any as string);
+    // this.question = JSON.parse(this.question as any as string);
```

Comment:
You fixed the data? if so remove the commented stuff? if not... what happened? :)

Reply:
I fixed the data for the test job. It has to be uploaded the way described in the docs. The old wikipedia data in the db will not work without the parse--not sure how that was uploaded.